Mogakco/Sources/Data/DataSources/Remote/RemoteUserDataSource.swift

```diff
@@ -20,13 +20,18 @@ struct RemoteUserDataSource: RemoteUserDataSourceProtocol {
         return provider.request(UserTarget.user(request))
     }
     
+    func allUsers() -> Observable<Documents<[UserResponseDTO]>> {
+        return provider.request(UserTarget.users)
+    }
+    
     func editProfile(id: String, request: EditProfileRequestDTO) -> Observable<UserResponseDTO> {
         return provider.request(UserTarget.editProfile(id, request))
     }
 }
 
 enum UserTarget {
     case user(UserRequestDTO)
+    case users
     case editProfile(String, EditProfileRequestDTO)
```

Comment:
```suggestion
    case user(UserRequestDTO)
    case allUsers
    case editProfile(String, EditProfileRequestDTO)
```

API 작성해주신 것 따라 네이밍을 따라가도 좋을 것 같아요! user, users는 구분이 잘 안갈 수 있을 것 같아요!

Reply:
넵 처음에 users였다가 allUsers로 바꿨는데 누락이 있었네요.
집 도착하면 수정하겠습니다!